src/libraries/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderCreateType.cs

```diff
@@ -28,6 +28,7 @@ public void CreateType(TypeAttributes attributes)
 
             TypeInfo typeInfo = type.CreateTypeInfo();
             Assert.Equal(typeInfo, createdType.GetTypeInfo());
+            Assert.NotNull(createdType);
```

Comment:
This better to be added before using the `createdType`, just after creating it

Reply:
Moved the Assertion to just after `createdType` creation.